src/content/en/tools/lighthouse/_toc.yaml

```diff
@@ -48,6 +48,8 @@ toc:
   path: /web/tools/lighthouse/audits/aspect-ratio
 - title: Document Does Not Have A Meta Description
   path: /web/tools/lighthouse/audits/description
+- title: Document Doesn't Use Legible Font Sizes
```

Comment:
Should we update the audit config to use "Does Not" for consistency with other audits?

Reply:
One way or the other, I was thinking about putting a PR in to make it consistent across reports

Reply:
Contracting it makes it feel a little more impersonal, which seems more in line with LH's voice